source/WindowScroller/WindowScroller.js

```diff
@@ -92,7 +113,7 @@ export default class WindowScroller extends Component {
   _onResizeWindow (event) {
     const { onResize } = this.props
 
-    const height = window.innerHeight || 0
+    const height = getHeight(this.scrollElement)
```

Comment:
We should probably call `this._updateDimensions()` here instead? That will handle cases where a resize affects top position (due to things like text wrapping) and it will also only call `setState` if the `height` has actually changed (eg not just a horizontal resize).

The `onResize` call should also be moved into this helper method for similar reasons (and to avoid duplication).

```js
_onResizeWindow (event) {
  this._updateDimensions()
}
```